examples/react-demo/src/App.tsx

```
@@ -0,0 +1,134 @@
+
+const client = new APIClient({
+  url: 'https://jsonplaceholder.typicode.com',
+});
+
+const routerDefinition = {
+  todos: {
+    get: client.route<void, Todo[]>({ method: 'GET', path: '/todos' }),
+    getById: client.route<void, Todo>({ method: 'GET', path: '/todos/:id' }),
+    create: client.route<TodoCreationRequest, Todo>({ method: 'POST', path: '/todos' }),
+    update: client.route<Todo, Todo>({ method: 'PUT', path: '/todos/:id' }),
+    delete: client.route<void, {}>({ method: 'DELETE', path: '/todos/:id' }),
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)